### DISCUSSION TITLE
What is the design system/ theme called?

### DISCUSSION THREAD

**Discussion by dit7ya** [pocketbase/pocketbase#732] [Q&A]
I really like the minimal grey-heavy style of pocketbase. I have also spotted the same design at https://www.automa.site/ .
So I am wondering if this is a well-known style but I cannot find anything.

**Comment by ganigeorgiev** [comment]
Sorry, I'm not aware if this type of design style has a name.

My main motivation with the current PocketBase frontend was to keep it as simple and generic as possible, almost without any branding so that it can be used as a rudimentary dashboard, no matter on what type of project you are working on.

For UI design inspirations I usually browse [dribbble](https://dribbble.com/), [screenlane](https://screenlane.com/) or [collectui](https://collectui.com/).

**Reply by edoversberger** [reply]
The UX is **fantastic**!
